fdbserver/RESTSimKmsVault.actor.cpp

```diff
@@ -236,6 +236,20 @@ void addBlobMetadaToResDoc(rapidjson::Document& doc, rapidjson::Value& blobDetai
 	key.SetString(BLOB_METADATA_LOCATIONS_TAG, doc.GetAllocator());
 	blobDetail.AddMember(key, locations, doc.GetAllocator());
 
+	// Add 'refreshAt'
```

Comment:
Can you help to add some comments about what are those new stuffs used for? Thanks!